code/workspaces/infrastructure-api/src/kubernetes/deploymentApi.js

```diff
@@ -23,7 +26,7 @@ const createOrReplace = (name, namespace, manifest) => (existingDeployment) => {
 };
 
 function getDeployment(name, namespace) {
-  return axios.get(`${getDeploymentUrl(namespace)}/${name}`)
+  return axios.get(`${getDeploymentUrl(namespace, name)}`)
```

Comment:
Is there a benefit to having the function call inside the string construct rather than just having the function call like `return axios.get(getDeploymentUrl(namespace, name))`.

Reply:
Good point, I'm too close to this code